misc/api-doc.ts

```diff
@@ -63,6 +83,32 @@ function isInternalMember(member) {
 	return false;
 }
 
+function isDecoratorOfType(decorator: Decorator, types: string[]) {
+	return (
+		isCallExpression(decorator.expression) &&
+		isIdentifier(decorator.expression.expression) &&
+		types.includes(decorator.expression.expression.text)
+	);
+}
+
+function getDecoratorOfType(node: Node, decoratorType: string): Decorator | null {
+	const decorators = canHaveDecorators(node) ? getDecorators(node) || [] : [];
+
+	for (const decorator of decorators) {
+		if (isCallExpression(decorator.expression) && isIdentifier(decorator.expression.expression)) {
```

Comment:
Couldn't we use `isDecoratorOfType` here rather than duplicating the logic?

Reply:
Will check that, thanks